src/devtools/client/debugger/src/components/PrimaryPanes/index.tsx

```diff
@@ -4,7 +4,10 @@
 
 //
 
+import clsx from "clsx";
```

Comment:
:hand: Can you switch from `clsx` to `classnames`?  Looks like we normally use `classnames` for this kind of step, and the only reason we have `clsx` in the codebase is because it's a dependency for `react-draggable`.

Reply:
@markerikson Thank you for your review!
I switched from `clsx` to `classnames`. 😀